app/services/vote_cache.rb

```diff
@@ -12,7 +12,7 @@ def initialize(user, votes)
   end
 
   def get_for(motion)
-    @votes_by_motion_id.fetch(motion.id) { Vote.new(motion: motion, user: user) }
+    @votes_by_motion_id[motion.id]
```

Comment:
This might break voting horribly in beta, not sure. I'm thinking the test suite will catch it if it does.